src/toxcore/messenger/packet/user_status.rs

```diff
@@ -0,0 +1,81 @@
+/*! UserStatus struct.
+*/
+
+use nom::le_u8;
+
+use crate::toxcore::binary_io::*;
+
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+#[repr(u8)]
```

Comment:
What is it for?

Reply:
It makes possible like this thing.
```
            gen_be_u8!(self.status as u8)
```

Reply:
Why is it not possible without repr?

https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=f61bd5f33844aecbf719174b567f1458